### DISCUSSION TITLE
Multiple api connections?

### DISCUSSION THREAD

**Discussion by ministryofsillywalks** [Hyundai-Kia-Connect/kia_uvo#680] [Q&A]
I already have evcc.io connected to the Hyundai api to control the charging of my car.
Can I hook this up in parallel to automate things like turning on the heater when its cold etc. or should only one service connect to the api to not mess anything up?

**Comment by fuatakgun** [comment]
your mobile app is connecting to Hyundai servers as we do in this integration. 

Number of active callers might get the attention of friends from Hyundai but we are not aware of any restriction.
